src/style-library/components/organisms/AppShellSkeleton.scss

```
@@ -0,0 +1,23 @@
+// Unlike the real `AppShell`, this placeholder is never wrapped in
+// `.kadence-blocks-style-library__shell` (which supplies the flex-column layout and the viewport
+// height) — it renders standalone at cold start (`StyleLibraryApp.js`, before `feed.isReady`) and,
+// reused as-is, inside `AppShell.js`'s `isBlocked` overlay (`.kadence-blocks-style-library__blocker`,
+// a `display: flex; align-items: center; justify-content: center` box with no width of its own).
+// Without sizing of its own, the skeleton shrink-wraps to its content and renders as a small,
+// centered box instead of filling the frame the way the spinner it replaced did.
+.kadence-blocks-style-library__shell-skeleton {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	// Fills the blocker overlay, which already gives `height: 100%` a real ancestor to resolve
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)